### ISSUE TITLE
Cop for excessive whitespace

### ISSUE THREAD

**Issue by jacob-carlborg** [bbatsov/rubocop#1888]
I'm not sure what the style guide says but I think there should be a cop for checking excessive whitespace. Example:

```ruby
a =            3
p a

```

Rubocop does not detect any offenses in the above code.

**Comment by bbatsov** [comment]
You can enable `Style/ExtraSpacing`. It's [disabled by default](https://github.com/bbatsov/rubocop/blob/master/config/disabled.yml).

**Comment by jacob-carlborg** [comment]
Awesome, thanks. BTW, why isn't this enabled by default?

**Comment by bbatsov** [comment]
Many people like to vertically align assignments.

On 12 May 2015 at 15:15, jacob-carlborg <notifications@github.com> wrote:

> Closed #1888 <https://github.com/bbatsov/rubocop/issues/1888>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bbatsov/rubocop/issues/1888#event-302737772>.
>



-- 
Best Regards,
Bozhidar Batsov

http://www.batsov.com

**Comment by jacob-carlborg** [comment]
Ah, good point.

**Comment by jacob-carlborg** [comment]
Sorry to bing this up again. Would it be possible to detect if the code is aligned or not? I'm mean, I don't think anyone wants their code to look like the example above.
